packages/grid/_modules_/grid/components/panel/filterPanel/GridFilterForm.tsx

```diff
@@ -34,7 +42,12 @@ const useUtilityClasses = (ownerState: OwnerState) => {
   const { classes } = ownerState;
 
   const slots = {
-    root: ['filterForm'],
+    root: ['filterForm', 'filterPanelFilterForm'],
```

Comment:
Why two classes on the root slot?

Reply:
Because renaming `'filterForm'` would be a breaking change, so to be consistent with the `'filterPannel'` prefix, I add `'filterPanelFilterForm'` and wait v6 to remove the `'filterForm'`

Reply:
Now that the prefix is "filterForm" it is not a problem anymore :)